Patches/The GiantRace/ThingDefs_Misc/GiantApparel.xml

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+  <Operation Class="PatchOperationSequence">
+	<operations>
+	  <li Class="PatchOperationFindMod">
+			
+		<mods><li>The GiantRace</li></mods>
+			
+		<match Class="PatchOperationSequence">
+		<operations>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="GI_GiantRags"]/statBases/StuffEffectMultiplierArmor</xpath>
+				<value>
+					<StuffEffectMultiplierArmor>2.5</StuffEffectMultiplierArmor>
```

Comment:
In general, the clothing should be markedly thicker--6 to 10 as the base, perhaps? With the relative increase in scale, clothing of thicknesses typical for human apparel is basically tissue paper to a giant.

Reply:
I'll double the thicknesses for the regular clothes/pauldrons. The rest are already mostly scaled up variants anyway (e.g. the brigadine isn't full plate, so it doesn't have 5mm, the helmet has closer to bascinet doubled stats than simple helmet etc), or shouldn't realy scale (the sash is a flag, and a flag isn't really any thicker, likewise with the booty).